npm/private/npm_translate_lock_state.bzl

```diff
@@ -108,8 +118,9 @@ def _init_common_labels(rctx, label_store):
     # pnpm-workspace.yaml file
     label_store.add_sibling("lock", "pnpm_workspace", PNPM_WORKSPACE_FILENAME)
 
-    # root package.json file
-    label_store.add_sibling("lock", "package_json_root", PACKAGE_JSON_FILENAME)
+    # root package.json file (if one exists)
+    if "//:package.json" in rctx.attr.data:
```

Comment:
not quite right; the root package.json will be a sibling to the pnpm-lock file so its Bazel package needs to be derived from where the lock file is located

Reply:
Updated